### ISSUE TITLE
Requiring sub-paths 

### ISSUE THREAD

**Issue by debopamsengupta** [scott113341/npm-install-version#34]
Hi @scott113341 ,

Would it be possible to add a parameter to use `niv.require` without replacing all `/` characters ?
This would allow us to require subpaths like `lodash@3.10.1/array/chunk` 

:)

**Comment by scott113341** [comment]
Good idea, this is a use case I didn't consider.  I'll look into this when I get the chance (going to be gone for the next two weeks).

**Comment by debopamsengupta** [comment]
Cheers ! 😄
